src/Altinn.App.Core/Features/Payment/Models/PaymentReceiver.cs

```diff
@@ -0,0 +1,11 @@
+namespace Altinn.App.Core.Features.Payment.Models;
+
+public class PaymentReceiver
+{
+    public string OrganisationNumber { get; set; }
+    public string Name { get; set; }
```

Comment:
Enten nullable eller required?
```suggestion
    public required string OrganisationNumber { get; set; }
    public required string Name { get; set; }
```